ui/shared/components/panel/LoadWorkspaceDataForm.jsx

```diff
@@ -148,7 +148,7 @@ const onProjectCreateSuccess = (responseJson) => {
 
 const formatAddDataUrl = ({ projectGuid }) => (`/api/project/${projectGuid}/add_workspace_data`)
 
-const onAddDataFromWorkspace = dispatch => (responseJson) => {
+const onAddDataFromWorkspace = responseJson => (dispatch) => {
```

Comment:
The existing code won't work. It couldn't be caught because we usually load data for newly created projects and never add data for current projects.